test/uk/gov/hmrc/traderservices/journey/TraderServicesFrontendFormatSpec.scala

```diff
@@ -114,7 +114,7 @@ class TraderServicesFrontendFormatSpec extends UnitSpec {
       validateJsonFormat(
         """{"state":"AnswerExportQuestionsContactInfo","properties":{"declarationDetails":{"epu":"123","entryNumber":"000000Z","entryDate":"2020-10-05"},
```

Comment:
Some really small things on this page but should be easily sorted:

Missing tests to validate json for **mandatory** vessel page (can largely copy&paste with some minor tweaks, missed on an earlier story but would be really appreciated if you got them in)

The existing export summary page test could probably do with adding in the contact details as well (as you've done for the import summary page test)